source/features/conversation-activity-filter.tsx

```diff
@@ -90,6 +90,14 @@ function processPage(): void {
 	}
 }
 
+function revealCollapsedComment(): void {
+	for (const item of select.all(`.js-timeline-item:is(.${collapsedClassName})`)) {
+		if (location.hash.startsWith('#issuecomment-') && select.exists(location.hash, item)) {
+			item.classList.remove(collapsedClassName);
+		}
+	}
```

Comment:
This should probably be the opposite:

1. Select the targeted item (if it exists)
2. Go back up the DOM (using `closest`) and remove the class on the parent `.js-timeline-item` element